torch/csrc/jit/codegen/cuda/scheduler/utils.cpp

```diff
@@ -16,11 +18,13 @@ namespace cuda {
 namespace scheduler_utils {
 // Merge all reduction to the right side and returns total number of
 // reduction axes
```

Comment:
Add comment on dont merge, allows us to place trivial reductions so they don't get merged into other axes.